packages/libs/react-components/src/components/Card/styles.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+
+import { styled } from '../../styles';
+
+import { VariantProps } from '@stitches/react';
+
+export const StyledCard = styled('div', {
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  padding: '$lg',
+  width: 'max-content',
+  height: 'max-content',
+  boxShadow: '$card',
+  borderRadius: '$lg',
+  variants: {
+    color: {
+      default: {
+        backgroundColor: '$defaultSurface',
+        color: '$defaultContrast',
+      },
+      negative: {
+        backgroundColor: '$neutral5',
+        color: '$neutral2',
+        h4: {
+          color: '$neutral2',
+        },
+      },
+    },
+    expand: {
+      true: { width: '100%', height: '100%' },
+      false: { width: 'max-content', height: 'max-content' },
+    },
+  },
+});
+
+export const StyledCardBody = styled('div', {
+  width: '100%',
+  fontFamily: '$main',
+  fontWeight: '$normal',
+  fontSize: '$base',
+  lineHeight: '$base',
+  marginTop: '$lg',
+});
+
+export const StyledCardFooter = styled('div', {
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'flex-start',
+  paddingTop: '$lg',
+  gap: '$sm',
+  width: '100%',
+});
+
+export type CardColor = VariantProps<typeof StyledCard>['color'];
+
+export const CardColors: { [key: string]: CardColor } = {
```

Comment:
You can look at other components as examples, but we have been separating the variants into separate objects so that there is one source of truth that is used for the variants and is exported and used in stories